documentation/src/docs/wire-protocol.md

```diff
@@ -102,3 +138,46 @@ After receiving the "Append Setup" reply, it performs the following:
 While this is happening, the server will be periodically sending it `DataAppended` replies acking messages. Note that there can be multiple "Appends Setup" for a given TCP connection. This allows a client to share a connection when producing to multiple Segments.
 
 A client can optimize its appending by specifying a large value in it's `AppendBlock` message, as the events inside of the block do not need to be processed individually.
+
+# Segment Attribute
+
+## GetSegmentAttribute - Request
+
+| **Field**    |**Datatype**  | **Description**     |
+|-------------|----------|------|
+| `RequestId`| Long| The client-generated _ID_ that identifies a client request.|
+|  `SegmentName`| String| |
+|  `attributeId`| UUID| |
+| `delegationToken`| String| This was added to perform _auth_. It is an opaque-to-the-client token provided by the Controller that says it's allowed to make this call.|
```

Comment:
@tom @shrids please provide definitions for Segment name and attributeid

Reply:
The segment to retrieve the attribute from.
The attribute to retrieve.

Reply:
Accepted suggestion